src/pymedphys/dicom/_level2/anonymise.py

```diff
@@ -213,6 +215,61 @@ def anonymise_dicom_dataset(ds, replace_values=True, delete_private_tags=True,
     return ds_anon
 
 
+def anonymise_dicom_file(dicom_filepath, overwrite_file=False, replace_values=True,
+                         keywords_to_leave_unchanged=None, delete_private_tags=True,
+                         delete_unknown_tags=None):
+
+    ds = pydicom.dcmread(dicom_filepath)
+
+    if overwrite_file:
+        dicom_anon_filepath = dicom_filepath
+    else:
+        dicom_anon_filepath = pjoin(dirname(dicom_filepath),
+                                    "{}.{}_Anonymised".format(basename(dicom_filepath)[0:2],
+                                                              ds.SOPInstanceUID))
+
+    ds_anon = anonymise_dicom_dataset(ds=ds,
+                                      replace_values=replace_values,
+                                      keywords_to_leave_unchanged=keywords_to_leave_unchanged,
+                                      delete_private_tags=delete_private_tags,
+                                      delete_unknown_tags=delete_unknown_tags,
+                                      copy_dataset=False)
+
+    ds_anon.save_as(dicom_anon_filepath)
+
+
+def anonymise_dicom_files_in_folder(folder, overwrite_files=False, replace_values=True,
+                                    keywords_to_leave_unchanged=None, delete_private_tags=True,
+                                    delete_unknown_tags=None):
+
+    for dicom_filepath in glob(folder + '/**/*.dcm', recursive=True):
```

Comment:
Does this globstring work for top level DICOM files?

Reply:
Yes. See example at the bottom of the [glob docs](https://docs.python.org/3/library/glob.html)

Reply:
though an `os.path.join()` is probably in order...

Reply:
> though an `os.path.join()` is probably in order...

glob can handle forward slashes on Windows just fine. So it's actually a non-issue in this case